gateway/src/node/client_handling/websocket/connection_handler/authenticated.rs

```diff
@@ -327,6 +327,13 @@ impl<R, S> AuthenticatedHandler<R, S> {
         }
     }
 
+    async fn handle_forget_me(
+        &mut self,
+        _also_from_stats: bool,
+    ) -> Result<ServerResponse, RequestHandlingError> {
+        unimplemented!()
```

Comment:
if we're leaving it unimplemented for now, maybe just log a `warn!` rather than crash the gateway itself : )

Reply:
That was just a placeholder :)